src/query/query_executor.c

```diff
@@ -23872,6 +23871,14 @@ qexec_evaluate_aggregates_optimize (THREAD_ENTRY * thread_p, AGGREGATE_TYPE * ag
 		  error = er_errid ();
 		  return (error == NO_ERROR ? ER_FAILED : error);
 		}
+
+	      spec->s_id.scan_stats.agg_index_name = NULL;
+	      error = heap_get_indexinfo_of_btid (thread_p, &ACCESS_SPEC_CLS_OID (spec), &agg_ptr->btid,
```

Comment:
Below is memory allocation with strdup. But it is not free.
```
(SCAN_ID * )->scan_stats.agg_index_name
```
It is being initialized with memset in the qexec_clear_access_spec_list function.
```
static int
qexec_clear_access_spec_list (THREAD_ENTRY * thread_p, XASL_NODE * xasl_p, ACCESS_SPEC_TYPE * list, bool is_final)
{
  ...
  for (p = list; p; p = p->next)
    {
      memset (&p->s_id.scan_stats, 0, sizeof (SCAN_STATS));
...
```

Reply:
I fixed next commit.